zerver/views/scheduled_messages.py

```diff
@@ -23,3 +26,40 @@ def get_scheduled_message(scheduled_message_id: int, user_profile: UserProfile)
         raise ResourceNotFoundError(_("Scheduled message does not exist"))
 
 
+def fetch_scheduled_messages(request: HttpRequest, user_profile: UserProfile) -> HttpResponse:
+    scheduled_messages = ScheduledMessage.objects.filter(
+        sender=user_profile, delivered=False, delivery_type=ScheduledMessage.SEND_LATER
+    )
+    scheduled_messages_to_dict: List[ScheduledMessageDict] = []
```

Comment:
I think `scheduled_message_dicts` would be a good name for this data structure.

Reply:
Okay, renamed.